code/ui/.storybook/main.ts

```diff
@@ -53,6 +54,11 @@ const config: StorybookConfig = {
     '@storybook/addon-designs',
     '@chromaui/addon-visual-tests',
   ],
+  build: {
+    test: {
+      disableBlocks: false,
+    },
+  },
```

Comment:
Needed because of:
https://github.com/storybookjs/storybook/blob/ab7fd6df9d0dd72526832acc9ca56f0ce99fa53d/code/ui/.storybook/preview.tsx#L141-L159

This breaks at runtime, because if blocks is replaced as a module by an empty object, that object (obviously) does not contain what is needed here.